sections/03-methods.md

```diff
@@ -0,0 +1,65 @@
+
+This model of disciplines as systems that interact in complex but operationally closed ways highlights what I see as the potential of practicing in multiple disciplines simultaneously: that asymmetric or apparently disparate activities can generate new insights in strangely context-specific forms. It is precisely the fact that methodological messages are skewed and transformed across systemic boundaries that can produce the possibility of unanticipated, non-schematic interactions and non-linear influences arising from the "gap" between disciplines or practices and their failure to directly communicate. Put simply, work in one discipline somehow ends up bending or altering another project asynchronously<sup>3</sup>. Insofar as "cybernetic methodologies draw out the virtuality correlated with actuality" (Clarke and Hansen 3), my proposed approach here perhaps also echoes the transversality of Felix Guattari, in that it shifts the methodological emphasis from "neatly tying everything up in a unidimensional manner... of simply discerning patterns" and instead towards establishing new assemblages made consistent by "virtual lines of bifurcation and differentiation" and "ontological heterogeneity" (Genosko 36-7).
```

Comment:
Highlights the potential or risks the potential? I guess it depends on the meaning of "operationally closed"